views/forms/UserProfileForm/UserProfileFormFields.jsx

```diff
@@ -36,6 +36,7 @@ const UserProfileFormFields = ({ control }) => {
         name="mail"
         size="small"
         fullWidth
+        disabled
```

Comment:
we just removed this in the last PR, are we sure we want to add it back?

Reply:
Yeah we had to un-disable it to fix a data issue in staging.